recipemd/main.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/python
```

Comment:
the shebang should be removed in this file as well (like you did in the other executables)